client/user/src/mealorder/meal-order-client-app.html

```diff
@@ -0,0 +1,313 @@
+<script src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/app-layout/app-drawer/app-drawer.html">
+<link rel="import"
+      href="../../bower_components/app-layout/app-drawer-layout/app-drawer-layout.html">
+<link rel="import" href="../../bower_components/app-layout/app-header/app-header.html">
+<link rel="import"
+      href="../../bower_components/app-layout/app-header-layout/app-header-layout.html">
+<link rel="import"
+      href="../../bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
+<link rel="import" href="../../bower_components/app-layout/app-toolbar/app-toolbar.html">
+<link rel="import" href="../../bower_components/app-route/app-location.html">
+<link rel="import" href="../../bower_components/app-route/app-route.html">
+<link rel="import" href="../../bower_components/iron-pages/iron-pages.html">
+<link rel="import" href="../../bower_components/iron-selector/iron-selector.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-tabs/paper-tabs.html">
+<link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="../../bower_components/iron-icons/maps-icons.html">
+<link rel="import" href="shell/redux-store.html">
+<link rel="import" href="menu/meal-order-menu-container.html">
+<link rel="import" href="menu/meal-order-drawer.html">
+<dom-module id="meal-order-client-app">
+    <template>
+        <style>
+            :host {
+                --app-primary-color: #4285f4;
+                --app-secondary-color: black;
+                display: block;
+            }
+
+            app-header {
+                background-color: #616161;
+                color: #fff;
+                right: 300px;
+            }
+
+            #main-title {
+                font-size: 1.375em;
+            }
+
+            app-drawer-layout {
+                background-color: #F2F2F2;
+            }
+
+            app-drawer-layout:not([narrow]) [drawer-toggle] {
+                display: none;
+            }
+
+            app-header app-toolbar .app-toolbar-container {
+                display: flex;
+                width: 100%;
+                align-items: center;
+                justify-content: space-between;
+            }
+
+            app-header .route-title a {
+                font-size: 0.8em;
+                text-decoration: none;
+                color: white;
+            }
+
+            app-header .route-title a:hover {
+                color: #179EFF;
+            }
+
+            app-header .history-icon {
+                display: none;
+                color: white;
+            }
+
+            app-header .app-toolbar-content {
+                font-size: 0.75em;
+            }
+
+            app-drawer.history {
```

Comment:
Let's not bind route to class. Create a class like `wide` and set class 'wide' to drawer if a specific route is activated.